README.md

```diff
@@ -2,25 +2,36 @@
 
 ## Objectives
 
-1. Identify the default application layout.
-2. Yield to view templates from a layout.
-3. Specify a custom layout in ActionController on a controller level using the `layout` macro and on the action level using the `render :layout => "custom"` option.
+1. Identify the default application layout
+2. Yield to view templates from a layout
+3. Specify a custom layout in ActionController on a controller level using the `layout` macro and on the action level using the `render :layout => "custom"` option
 4. Use :layout => false to shut off the layout and only render the view
 
-## Outline
+## The Default Layout
 
-Give them a rails app with no layout and no controllers (besides applicationcontroller). There should be working models though. Pretty much making a shopping cart.
+1. Generate a new controller called `StaticController`, by going onto the command line and typing `rails generate controller Static`
+2. Create a new action in `StaticController` called `home`
+3. Create a template file for the `home` action at app/views/static/home.html.erb, and insert an H2 containing the text **Welcome to Flatiron Widgets**
+3. Create a default application layout at app/views/layouts/application.html.erb and add an H1 to it containing the text **Flatiron Widgets Store**, this is for the main site's welcome
+4. Remember to add the `yield` expression to tell it where the action template should be rendered
 
-Default Layout
+## Custom Layouts for a Controller
 
-1. Create a StaticController
-2. Create a default application layout and put an H1 Flatiron Widgets Store for the site's main welcome.
-3. Static#home should render that layout and the view for Static#home (static/home.html.erb) which has an H2 Welcome to Flatiron Widgets.
+1. Create a new controller called `StoreAdminController`. You can easily generate this controller by typing `rails generate controller StoreAdmin`
+2. Create a new layout file at layouts/admin.html.erb and add an H1 to it containing the text **Flatiron Widgets: Admin**
+3. Create a new action in `StoreAdminController` called `home`
+4. Now you need to create a template for the `home` action at app/views/store_admin/home.html.erb, and in here insert an H2 containing the text **Welcome Flatiron Admin**
+5. At this point the store_admin#home action will be using the default application layout. Use the `layout` controller macro in the `StoreAdminController` to make it use the layout you created at layouts/admin.html.erb
```

Comment:
things like this just tell them to use a different layout. They should go back to the reading